apps/lib/cmp_mock_srv.c

```diff
@@ -391,6 +408,46 @@ static OSSL_CMP_PKISI *process_rr(OSSL_CMP_SRV_CTX *srv_ctx,
     return OSSL_CMP_PKISI_dup(ctx->statusOut);
 }
 
+/* return -1 for error, 0 for no update available */
+static int check_client_crl(const STACK_OF(OSSL_CMP_CRLSTATUS) *crlStatusList,
+                            const X509_CRL *crl)
+{
+    OSSL_CMP_CRLSTATUS *crlstatus;
```

Comment:
```suggestion
    OSSL_CMP_CRLSTATUS *crlstatus = NULL;
```

Reply:
This is assigned a few lines below. Unnecessary.